view/frontend/templates/breadcheckout/view.phtml

```diff
@@ -16,7 +16,8 @@
     {
         "*": {
              "Bread_BreadCheckout/js/product/button": {
-                "buttonId": "#bread-checkout-btn"
+                "buttonId": "#bread-checkout-btn, #button-prevent",
+                "productConfigAndOptionsSelector": ".product-options-wrapper [aria-required=\"true\"]"
```

Comment:
oh good call moving it here. could you also add something about this only grabbing required options in the variable name?